components/mpas-ocean/src/mode_forward/mpas_ocn_time_integration_split.F

```diff
@@ -1753,8 +1789,26 @@ subroutine ocn_time_integrator_split(domain, dt)!{{{
             end if
          end do
          call mpas_timer_stop("se halo tracers")
+         if(config_enable_nonhydrostatic_mode) call ocn_tend_vvel(tendPool, statePool, &
+                          forcingPool, meshPool, 2, dt)
 
          call mpas_timer_start('se loop fini')
+         call mpas_pool_get_array(tracersPool, 'activeTracers', &
+                                                tracersGroupCur, 1)
+         call mpas_pool_get_array(tracersPool, 'activeTracers', &
+                                                tracersGroupNew, 2)
+         call mpas_pool_get_array(statePool,   'normalVelocity', &
+                                                normalVelocityCur, 1)
+         call mpas_pool_get_array(tracersTendPool,'activeTracersTend', &
+                                                   activeTracersTend)
```

Comment:
any ideas why these tracer arrays show up in this PR?  They should be unrelated to any non hydro code